### PR TITLE
ANAP-59 Upgrade dependencies version

### PR DESCRIPTION
**Pull request by Tronok**
Upgrade dependencies versions and  fix backward incompatible issues

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,720 +2,678 @@
 # yarn lockfile v1
 
 
-"7zip-bin@~4.0.2":
-  version "4.0.2"
-  resolved "https://registry.yarnpkg.com/7zip-bin/-/7zip-bin-4.0.2.tgz#6abbdc22f33cab742053777a26db2e25ca527179"
-  integrity sha512-XtGk+IF57pr852UK1AhQJXqmm1WmSgS5uISL+LPs0z/iAxXouMvdlLJrHPeukP6gd7yR2rDTMSMkHNODgwIq7A==
+"7zip-bin@~5.0.3":
+  version "5.0.3"
+  resolved "https://nexus.global.picnicinternational.com/repository/npm-public/7zip-bin/-/7zip-bin-5.0.3.tgz#bc5b5532ecafd923a61f2fb097e3b108c0106a3f"
```
Comment (Stephan202) [inline yarn.lock:7]: ^ This is a public repo, so we must not use the private Nexus deployment.

**Review by Stephan202 — APPROVED** [review]
@japborst or @larslockefeer perhaps one of you can review this.

File: package.json [hunk 2]
```diff
@@ -71,27 +71,27 @@
     "devDependencies": {
         "chai": "*",
         "chai-as-promised": "^7.1.1",
-        "chai-http": "^4.2.0",
-        "electron": "^2.0.10",
-        "electron-builder": "^20.28.4",
-        "husky": "^0.14.3",
+        "chai-http": "^4.3.0",
+        "electron": "^9.3.0",
```
Comment (japborst) [inline package.json:75]: I suspect this will break the app. Build also fails, have you tested this locally?

File: test/main.js [hunk 3]
```diff
@@ -112,12 +112,13 @@ describe("NepSnowplow", function() {
...
 
         it("starts without errors", function() {
-            return this.app.client
-                .waitUntilWindowLoaded()
-                .getRenderProcessLogs()
-                .then(function(logs) {
-                    logs.filter((log) => log.level === "SEVERE").should.have.lengthOf(0);
-                });
+             return this.app.client
+                .waitUntilWindowLoaded().then(()=>{
```
Comment (Stephan202) [inline test/main.js:116]: I suspect running the formatter will add some spacing here.

File: package.json (hunk 2 above)
Comment (Tronok) [reply package.json:75]: @japborst yes, it works. I've updated tests and manually tested it. But however I cannot make the build pass due to issues with nodejs version. Maybe could you help with that?

**Review by japborst — APPROVED** [review]
Thanks @Tronok !

**Comment by Stephan202** [comment]
One open point, I think: reapplication of the formatter.

**Comment by Stephan202** [comment]
Formatting was deferred to a separate PR, see #10.

**Pull request merged**